azurelinuxagent/ga/monitor.py

```diff
@@ -177,11 +180,35 @@ def collect_and_send_events(self):
         except ProtocolError as e:
             logger.error("{0}", e)
 
+    def send_daily_heartbeat(self):
+        """
+        Send the daily heartbeat containing the installed and current agent versions.
+        Send once during the first 30 minutes of the day.
+        """
+        if datetime.datetime.time() <= datetime.time(0, minute=30):
```

Comment:
@hglkrijger I wasn't...but maybe we should be. I'll re-evaluate emitting once-per-day based on a timestamp.